dev-env.yaml

```diff
@@ -31,11 +31,12 @@ services:
     ports:
       - "5432:5432"
     environment:
-      POSTGRES_DB: ${postgres_db_name}
-      POSTGRES_HOST: ${postgres_fqdn}
-      POSTGRES_USER:  ${postgres_user}
-      POSTGRES_PASSWORD: ${postgres_password}
-      sslmode: require
+      POSTGRES_DB: ${postgres_db_name:-reportvision}
```

Comment:
Does it matter if the `dev-env.yaml` is using `:-name` and the `application.yaml` file is not?

Reply:
It seems one is springboot synthax and the other is bash synthax. app.yaml is being used locally so it should work as is.